tau-ml-env.yaml

```diff
@@ -328,6 +328,7 @@ dependencies:
   - zstandard=0.15.2=py38h497a2fe_0
   - zstd=1.4.9=ha95c52a_0
   - pip:
+    - law
```

Comment:
please, specify the exact version as it is done for all other packages.

Reply:
done 
https://github.com/lguzzi/TauMLTools/commit/7646a23680c54bc82cec3ccb37f5aa086e7c01c7